Description: Don't try to be clever detectecting where
 php_session.h is, default works.
Author: Christian Perrier <bubulle@debian.org>
Forwarded: no

---
 memcache-3.0.6/config9.m4 |   27 +--------------------------
 1 file changed, 1 insertion(+), 26 deletions(-)

--- a/memcache-3.0.6/config9.m4
+++ b/memcache-3.0.6/config9.m4
@@ -62,37 +62,12 @@
     PHP_ADD_INCLUDE($PHP_ZLIB_INCDIR)
   fi
  
-  if test "$PHP_MEMCACHE_SESSION" != "no"; then 
-	AC_MSG_CHECKING([for session includes])
-    session_inc_path=""
-
-    if test -f "$abs_srcdir/include/php/ext/session/php_session.h"; then
-      session_inc_path="$abs_srcdir/include/php"
-    elif test -f "$abs_srcdir/ext/session/php_session.h"; then
-      session_inc_path="$abs_srcdir"
-    elif test -f "$phpincludedir/ext/session/php_session.h"; then
-      session_inc_path="$phpincludedir"
-    else
-      for i in php php4 php5 php6; do
-        if test -f "$prefix/include/$i/ext/session/php_session.h"; then
-          session_inc_path="$prefix/include/$i"
-        fi
-      done
-    fi
-
-    if test "$session_inc_path" = ""; then
-      AC_MSG_ERROR([Cannot find php_session.h])
-    else
-      AC_MSG_RESULT([$session_inc_path])
-    fi
-  fi
-
   AC_MSG_CHECKING([for memcache session support])
   if test "$PHP_MEMCACHE_SESSION" != "no"; then
     AC_MSG_RESULT([enabled])
     AC_DEFINE(HAVE_MEMCACHE_SESSION,1,[Whether memcache session handler is enabled])
     AC_DEFINE(HAVE_MEMCACHE,1,[Whether you want memcache support])
-    PHP_NEW_EXTENSION(memcache, memcache.c memcache_pool.c memcache_queue.c memcache_ascii_protocol.c memcache_binary_protocol.c memcache_standard_hash.c memcache_consistent_hash.c memcache_session.c, $ext_shared,,-I$session_inc_path)
+    PHP_NEW_EXTENSION(memcache, memcache.c memcache_pool.c memcache_queue.c memcache_ascii_protocol.c memcache_binary_protocol.c memcache_standard_hash.c memcache_consistent_hash.c memcache_session.c, $ext_shared,,)
     ifdef([PHP_ADD_EXTENSION_DEP],
     [
       PHP_ADD_EXTENSION_DEP(memcache, session)
